helm_deploy/secrets-example.yaml

```diff
@@ -2,3 +2,4 @@
 secrets:
   INTERVENTIONS_SERVICE_CLIENT_ID: clientid
```

Comment:
Totally not related: `SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_INTERVENTIONSCLIENT_CLIENTID` is the one we may need, I guess? 🤷‍♂️